app/src/androidTest/java/com/github/sdp_begreen/begreen/firebase/FirebaseDBTest.kt

```diff
@@ -204,4 +204,13 @@ class FirebaseDBTest {
             ))
         }
     }
+
+    @Test
+    fun getAllUsersReturnTheSameNumberOfUsers() {
```

Comment:
I suggest you to rename this test : `getAllUsersReturnNonEmptyListOfUsers` or something like that since it corresponds more to what the test actually tests.

Reply:
Oh yeah sure! In fact I wasn't able to find the exact number of user in the emulators of the CI and it could change so I modified to >0 but forgot to change the name! Thx